app/helpers/batch_connect/sessions_helper.rb

```diff
@@ -105,17 +105,17 @@ def id(session)
 
   def status(session)
     if session.starting?
-      "Starting"
+      t('dashboard.bc.sessions.stats.status.starting')
     elsif session.running?
-      "Running"
+      t('dashboard.bc.sessions.stats.status.running')
     elsif session.queued?
-      "Queued"
+      t('dashboard.bc.sessions.stats.status.queued')
     elsif session.held?
-      "Held"
+      t('dashboard.bc.sessions.stats.status.held')
     elsif session.suspended?
-      "Suspended"
+      t('dashboard.bc.sessions.stats.status.suspended')
     else
-      "Undetermined"
+      t('dashboard.bc.sessions.stats.status.undetermined')
```

Comment:
Lets leave the status values un-localizable at this time. I would like to address this in a different way (by possibly introducing internationalization to ood_core).